src/components/test-summary/test-summary.ts

```diff
@@ -1,41 +1,33 @@
 import { Component } from '@angular/core';
 import { IFaultSummary, FaultTitle } from './interfaces/IFaultSummary';
```

Comment:
Can you prefix FaultTitle with an i, so it's easier to see that it's an Interface.

Reply:
FaultTitle is an enum, not an interface. I could split it out into its own file so it's not in a file with other interfaces if you want?

Reply:
oh yeah. Erm..might make sense in another folder/file, or just in the test-summary.ts - if it's only used there.